src/daria/image/image.py

```diff
@@ -49,38 +50,68 @@ class Image:
 
     def __init__(
         self,
-        img: np.ndarray,
-        origo: Union[np.ndarray, list[float]] = np.array([0, 0]),
-        width: float = 1,
-        height: float = 1,
-        depth: float = 0,
-        dim: int = 2,
-        read_metadata_from_file: bool = False,
-        metadata_path: Optional[str] = None,
-        # TODO Have to rethink the use of parameters, and possibly switch to
-        # a config file instead, that is shared among a full suite of images.
-        # It should contain information on the correction routines, necessary.
-        # It should include physical information: width, height
-        # It should include the format of timestamps included in the exifdata
+        img: Union[np.ndarray, str],
+        metadata: Optional[dict] = None,
+        curvature_correction: Optional[da.CurvatureCorrection] = None,
+        color_correction: Optional[da.ColorCorrection] = None,
```

Comment:
Again more an observation than a comment (I just have to get used to the new changes I believe) - having to pass the curvature and color correction from outside (instead of creating them inside of `daria.Image` from a config file) will require more effort from the user, in particular when working with just a single image and not a series. For the latter, it feels quite natural that the correction objects are centralized objects, as also commented on above. To me this is very much OK, but I have to say, I was surprised at first, when I saw these changes, as I suspected a decreased and increased need of user-input here.

Reply:
We could do it like you say, I was just afraid that too much will be "hidden" inside the initialization of da.Image. Now, at least it is very clear that we would like to apply corrections to the images that are initialized this way. We could of course also just accept paths to config files for the corrections in the input of __init__. Then we do not have to define the correction before the Image. I am not sure what I really prefer. It is anyways only one/two lines more of user code, and it is in principle very clear what is happening at these lines.

Reply:
Agree. At this stage, I find this approach the best. Feel free to resolve.